mrtt-ui/src/components/SocioeconomicAndGovernance/SocioeconomicAndGovernanceStatusAndOutcomesForm.js

```diff
@@ -0,0 +1,233 @@
+import { useState } from 'react'
+import { useParams } from 'react-router-dom'
+import { yupResolver } from '@hookform/resolvers/yup'
+import * as yup from 'yup'
+import axios from 'axios'
+import { toast } from 'react-toastify'
+import { AdapterDateFns } from '@mui/x-date-pickers/AdapterDateFns'
+import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider'
+import { MobileDatePicker } from '@mui/x-date-pickers/MobileDatePicker'
+import { Controller, useForm } from 'react-hook-form'
+import { MenuItem, Stack, TextField } from '@mui/material'
+
+import { Form, FormPageHeader, FormQuestionDiv, StickyFormLabel } from '../../styles/forms'
+import QuestionNav from '../QuestionNav'
+import useSiteInfo from '../../library/useSiteInfo'
+import language from '../../language'
+import { ContentWrapper } from '../../styles/containers'
+import { questionMapping } from '../../data/questionMapping'
+import { ErrorText, PageSubtitle, PageTitle } from '../../styles/typography'
+import { mapDataForApi } from '../../library/mapDataForApi'
+import { socioeconomicGovernanceStatusOutcomes as questions } from '../../data/questions'
+import LoadingIndicator from '../LoadingIndicator'
+import FormValidationMessageIfErrors from '../FormValidationMessageIfErrors'
+import useInitializeQuestionMappedForm from '../../library/useInitializeQuestionMappedForm'
+import CheckboxGroupWithLabelAndController from '../CheckboxGroupWithLabelAndController'
+import { multiselectWithOtherValidationNoMinimum } from '../../validation/multiSelectWithOther'
+
+const SocioeconomicAndGovernanceStatusAndOutcomesForm = () => {
+  const { site_name } = useSiteInfo()
+  const validationSchema = yup.object({
+    dateOfOutcomesAssessment: yup.string().nullable(),
+    changeInGovernance: yup.string(),
+    currentGovenance: multiselectWithOtherValidationNoMinimum,
+    changeInTenureArrangement: yup.string(),
+    currentLandOwnership: multiselectWithOtherValidationNoMinimum,
+    rightsToLandInLaw: yup.string(),
+    achievementOfSocioeconomicAims: yup.string()
+  })
+  const reactHookFormInstance = useForm({
+    defaultValues: {
+      currentGovenance: { selectedValues: [], otherValue: undefined },
```

Comment:
is `otherValue` needed? Doesnt look like this question needs it.

Reply:
yeah good call, will remove `otherValue`